### DISCUSSION TITLE
Embedded .pdf

### DISCUSSION THREAD

**Discussion by CarolusIII** [oleeskild/obsidian-digital-garden#350] [General]
Good morning,
I was trying to publish a note that had a embedded .pdf, but nothing I tried worked. The webpage does show the embed link, but does not display the actual pdf content underneath the embed link as one would expect.
Am I doing somethings wrong or is it a Vercel build problem.
What are the expectations fro finding a solution.

By the way, DG is a great plugin, adds a lot of value to using Obsidian.
Thanks for all that.

**Comment by chaosraine** [comment]
Did the link work for you, or did it just display as text? I just came across the same thing. Will end up just throwing my pdfs in google drive and linking to them, because embedding isn't that important to me, but I suppose you could do an iframe or something as a workaround?

**Reply by CarolusIII** [reply]
No, i tried everything I could think of, but apart from a link I could not produce more. So I ended up placing all my .pdf's on GDrive and then linking my Obsidian texts to the external pdf's.
Actually I am satisfied it is not a bad solution, nothing special, but it works. Just check it out if you want: [https://my-life-bxir.vercel.app/home/persoonlijk/cv-and-levensloop/.](https://my-life-bxir.vercel.app/my-life/persoonlijk/cv-and-levensloop/)
Now I am struggling with my fonts and CSS and how to modify those. Life goes on.
Cheers.

**Comment by yone-house** [comment]
I did this using an <iframe> with a pdf file stored in Google Drive.

Just set up a public sharing link, which should be something like "drive.google.com/file/d/[FILE ID]/view?usp=drive_link," and create an iframe like:

`<iframe src="drive.google.com/file/d/[FILE ID]/preview"></iframe>`

Add whatever other styles or [iframe attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/iframe) you want, and it should be in there.

There are plenty of other sources you can store a PDF in and use for the <iframe>, I just happened to use Drive.
